src/lib/chunked_pedersen_lib/pedersen.ml

```diff
@@ -136,8 +136,9 @@ struct
 
     let update_fold (t : t) (fold : bool Triple.t Fold.t) =
       O1trace.measure "pedersen fold" (fun () ->
-          let max_num_params = Array.length params in
-          (* As much space as we could need: we can only have up to [length params] triples before we overflow that, and each triple is packed into a single byte *)
+          let params_forced = Lazy.force params in
```

Comment:
Can this just be `params` instead of `params_forced`? I'm not sure this detail improves clarity for the rest of the function.